src/mango/test/05-index-selection-test.py

```diff
@@ -183,6 +183,7 @@ def test_prefer_use_index_over_optimal_index(self):
 
     # This doc will not be saved given the new ddoc validation code
     # in couch_mrview
+    @unittest.skip("need to add couch_mrview:validate_ddoc_fields")
```

Comment:
Disabling tests doesn't seem like the answer here. This should be part of the PR one way or another.

Reply:
Fixed